scripts/deploy/deploy.ts

```diff
@@ -67,11 +76,11 @@ export async function main(
     await loanCore.deployed();
     console.log("LoanCore deployed to:", loanCore.address);
 
-    // Grant correct permissions for promissory note
-    // Giving to user to call PromissoryNote functions directly
-    for (const note of [borrowerNote, lenderNote]) {
-        await note.connect(admin).initialize(loanCore.address);
-    }
+    // // Grant correct permissions for promissory note
```

Comment:
Can just remove this now?

Reply:
Also on line 58 and 70, some blank lines after `await borrowerNote.deployed()` and `await lenderNote.deployed()` would be nice and improve readability (as well as semicolons at the end of the lines).